controllers/internal/hostdefiner/hostdefiner.go

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2022 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package hostdefiner
+
+import (
+	"fmt"
+
+	csiv1 "github.com/IBM/ibm-block-csi-operator/api/v1"
+	"github.com/IBM/ibm-block-csi-operator/controllers/internal/common"
+	"github.com/IBM/ibm-block-csi-operator/pkg/config"
+	csiversion "github.com/IBM/ibm-block-csi-operator/version"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+type HostDefiner struct {
+	*csiv1.HostDefiner
+}
+
+func New(c *csiv1.HostDefiner) *HostDefiner {
+	return &HostDefiner{
+		HostDefiner: c,
```

Comment:
`c` is already used (internally) for `IBMBlockCSI` (because `CSI` starts with c?).
consider using something different (e.g. `h`, `hd`, etc...)

Reply:
done - #271